wallet/src/cli.rs

```diff
@@ -103,6 +109,20 @@ pub enum Command {
     ShowTimestamp,
 }
 
+#[derive(Debug, Args)]
+pub struct MintCoinArgs {
+
+    /// Pass the amount to be minted, if not passed 100 coins will be minted 
+    #[arg(long, short, verbatim_doc_comment, action = Append)]
+    pub amount: Option<u128>,
```

Comment:
Remove the option, and use clap to provide a default value. Then clap will also be responsible for showing the default in the help pages so you do not need to write it manually.